diego_villanueva/week04/movies/app.rb

```diff
@@ -0,0 +1,33 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require 'httparty'
+
+get '/' do
+  erb :index, layout: :main_layout
+end
+
+
+get '/movie' do
+  movie_title = params[:movie_title]
+  response = HTTParty.get( 'http://omdbapi.com/?t=' + movie_title )
+  @movie_title = response["Title"]
+  @movie_year = response["Year"]
+  @movie_actors = response["Actors"]
+  @movie_image = response["Poster"]
+  erb :movie, layout: :main_layout
+end
```

Comment:
Probably easier to just make a `@movie_details` variable from the response and pass the whole thing to your template, instead of lots of specific variables like this.